data/json/effects.json

```
@@ -2809,6 +2810,74 @@
     "max_duration": 14515200,
     "dur_add_perc": 50
   },
+  {
+    "//": "For player, indistinguishable from pre_conjunctivitis_bacterial, but separate here to allow for separate resistances from antibiotics.",
+    "type": "effect_type",
+    "id": "pre_conjunctivitis_viral",
+    "show_in_info": false,
+    "immune_flags": [ "NO_DISEASE", "SEESLEEP" ],
+    "blood_analysis_description": "Viral Infection",
+    "max_duration": "71 h"
+  },
+  {
+    "//": "For player, indistinguishable from pre_conjunctivitis_viral, but separate here to allow for separate resistances from antibiotics.",
+    "type": "effect_type",
+    "id": "pre_conjunctivitis_bacterial",
+    "show_in_info": false,
+    "immune_flags": [ "NO_DISEASE", "SEESLEEP" ],
```

```suggestion
    "immune_flags": [ "INFECTION_IMMUNE", "SEESLEEP" ],
```
Thoughts on splitting them this way or just including INFECTION_IMMUNE for bacterial infection?

---

They should probably both be infection_immune, since it's an infection of the dermis akin to staph or something. Sometimes it even is staph.